src/components/plots/styling/volcano/ThresholdsGuidesEditor.jsx

```diff
@@ -78,13 +81,10 @@ const ThresholdsGuidesEditor = (props) => {
             <Space>
               <InputNumber
                 min={0}
-                defaultValue={config.negLogpValueThreshold}
+                value={config.negLogpValueThreshold}
```

Comment:
Does this work well? It updates the render fast? Or does it lag waiting for the dispatch and redux to update?